conf/composites.conf

```diff
@@ -103,16 +103,19 @@ composites {
         expression = "(HAS_X_POS | HAS_PHPMAILER_SIG | HAS_X_PHP_SCRIPT) & (SUBJECT_ENDS_QUESTION | SUBJECT_ENDS_EXCLAIM | MANY_INVISIBLE_PARTS)";
         description = "Message was generated by PHP script and contains some spam indicators";
         score = 1.0;
+        policy = "leave";
     }
     PHISH_EMOTION {
         expression = "(PHISHING | DBL_PHISH | PHISHED_OPENPHISH | PHISHED_PHISHTANK) & (SUBJECT_ENDS_QUESTION | SUBJECT_ENDS_EXCLAIM)";
         description = "Phish message with subject trying to address users emotion";
         score = 2.0;
```

Comment:
Probably the score should be adjusted as you leave symbols scores. `PHISHING` score is dynamic and in many cases can be close to zero. Adding significant score to it can lead to false positives.

Reply:
Thanks for the comment; I am thinking about adding other phishy symbols to it to avoid FPs.

Is 1.0 low enough for now?

Reply:
Yes, I think so.